calcom/cal.com
# Fixing already installed apps & link to them

## What does this PR do?

Fixing already installed apps that were not showing a disabled "Installed" button as before.
<kbd><img width="1139" alt="image" src="https://user-images.githubusercontent.com/467258/180470027-4aca15bc-d9ce-46e0-8154-353f61e92637.png"></kbd>

Also took the opportunity to add a link to the already installed apps to their own page, in case the user wants to install it again if the app allows it.
<kbd><img width="1157" alt="image" src="https://user-images.githubusercontent.com/467258/180470857-97c72a4c-1782-4fbe-9514-83011e861dd3.png"></kbd>

**Environment**: Staging(main branch) / Production

## Type of change


## How should this be tested?

For example, install Zapier or HubSpot, both apps without the possibility to install multiple instances. It should show in their app page that the app is already installed. And in the Installed Apps page, you can go to each app page through its title.
